sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/Image.java

```diff
@@ -96,4 +97,20 @@ public void setImageType(@NonNull ImageType imageType) {
     public ImageType getImageType() {
         return (ImageType) getObject(ImageType.class, KEY_IMAGE_TYPE);
     }
+
+    /**
+     * set whether the image is a template

```

Comment:
Can we add a description on what it means to be a template image? Also fix the javadoc on these methods to be actual sentences (start with a capital add punctuation). I see the other javadocs for the methods have the same issue, optional to cleanup those as well.